src/modules/VirtConfig.rb

```diff
@@ -108,10 +108,10 @@ def isTumbleweed
       end
       false
     end
-    def isSLED
-      Builtins.y2milestone("Checking to see if this is SLED ...")
+    def isSLES
+      Builtins.y2milestone("Checking to see if this is SLES ...")
       distro = OSRelease.ReleaseName
-      if distro.include? "SLED"
+      if distro.include? "SLES"
         Builtins.y2milestone("Platform is %1", distro)
         return true
```

Comment:
NP: The function ends here, so strictly speaking, 'return' is not necessary; you can just write

    false

(another style issue)